src/deepsparse/server/cli.py

```diff
@@ -27,7 +27,7 @@
 import click
 import yaml
 
-from deepsparse.pipeline import SupportedTasks
+from deepsparse.base_pipeline import SupportedTasks
```

Comment:
i think `SupportedTasks` would still be useful to bubble-up to the `pipeline` level import

Reply:
Any specific reason for this? I guess similar idea to below, since we've moved the task constructor getter and register functions to the `BasePipeline`, thought it made sense to keep this as the `BasePipeline` level.